tui/src/hint.rs

```diff
@@ -64,7 +62,7 @@ impl Shortcut {
         Self {
             key_sequences: key_sequences
                 .iter()
-                .map(|s| Span::styled(Cow::<'static, str>::Borrowed(s), Style::default().bold()))
+                .map(|s| Span::styled(*s, Style::default().bold()))
```

Comment:
```suggestion
                .map(|s| Span::styled(Cow::<'_, str>::Borrowed(s) Style::default().bold()))
```

This isn't an optimization. There's no reason to deref the str slice, just use a `Cow` to prevent unnecessary copies. It's going to use a `Cow` under the hood, anyways. `Cow` is a part of `std`, so there's no benefit to removing the import either.

Reply:
Ahh I see.